async_plus/tasks.py

```diff
@@ -4,12 +4,25 @@
 import logging
 
 
-__all__ = ['launch_watched', 'task_scope']
+__all__ = ['try_gather', 'launch_watched', 'task_scope']
 
 
 logger = logging.getLogger(__name__)
 
 
+async def try_gather(*futs):
+    """ Safe version of gather that doesn't leak tasks """
```

Comment:
nit

| Code | Description                                       |
|------|---------------------------------------------------|
| D210 | No whitespaces allowed surrounding docstring text |
| D400 | First line should end with a period               |